common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1075,6 +1075,43 @@ div.problem {
     font-size: 0.9em;
   }
 
+  .notification {
+    margin-top: $baseline / 2;
+    padding: ($baseline * 0.4) ($baseline / 2);
+    min-width: auto;
+    &.correct,
```

Comment:
Nit: perhaps we should make these both class "success", and the incorrect class "error"? So the classes would be success, error, info, and warning? It comes down to how much we want to make the class names general vs. tied to the use cases.